cmd/gitserver/internal/git/gitcli/refs_test.go

```diff
@@ -399,6 +400,8 @@ func TestGitCLIBackend_ListRefs_GoroutineLeak(t *testing.T) {
 	// Don't complete reading all the output, instead, bail and close the reader.
 	require.NoError(t, it.Close())
 
+	time.Sleep(time.Millisecond)
```

Comment:
This makes the test a little more resilient to flakes because it gives the associated goroutines one more chance to get scheduled.

Reply:
Why not something more than enough, like 500 ms?

```suggestion
time.Sleep(500 * time.Millisecond)
```

Reply:
It would make the tests much slower to run, they're pretty fast right now and 500ms is quite a lot :/ If this flakes, we can go to 10ms or so first, and see? But so far I haven't been able to get it to flake locally.